docs/reference/UDS Core/IdAM/upgrading-versions.md

```diff
@@ -10,65 +10,99 @@ This doc contains important information for upgrading uds-identity-config versio
+         - Click `Save`
+   - Configure the UDS Client Policy
+      - Go to `Realm Settings` > `Client Policies` > `Profiles`
+         - Click `Create Client Profile`
+               - Name: `uds-client-profile`
+               - Description: `UDS Client Profile`
+               - Click `Save`
+         - Click `Add Executor`
+               - Select `uds-operator-permissions`
+               - Click `Add`
+      - Go to `Realm Settings` > `Client Policies` > `Policies`
+         - Click `Create client policy`
+               - Name: `uds-client-policy`
+               - Description: `UDS Client Policy`
+         - Click `Add condition`
+         - Select `any-client`
+         - Click `Add`
+         - Click `Add client profile`
+         - Select `uds-client-profile`
+         - Click `Add` (there is a glitch in the UI where it seems all the profiles are selected, but only the selected one is actually chosen)
+   - Configure the Client Credentials Authentication Flow
+      - Go to `Authentication` > `Flows`
+         - Click `clients`
+               - Click `Actions` > `Duplicate`
+                  - Name: `UDS Client Credentials`
+                  - Description `UDS Client Credentials`
+                  - Click `Duplicate`
+         - Go to `Authentication` > `UDS Client Credentials`
+               - Click `Add Step`
+                  - Select `Client Id and Kubernetes Secret`
+                  - Click `Add`
+               - Select `Requirement` and set it to `Alternative`
+         - Go to `Authentication`, select three dots on the right side of the panel for `UDS Client Credentials` and select `Bind flows`
+               - Select `Client authentication flow`
+               - Click `Save`
+   - Verify that everything is configured correctly
+      - Deploy a new package or update the existing one
+      - Check UDS Operator logs and verify if there are no errors
+         - Use `uds zarf tools kubectl logs deploy/pepr-uds-core-watcher -n pepr-system | grep "Client Credentials Keycloak Client is available"` command to verify if the UDS Operator uses the Client Credentials flow.
 
 After introducing the above changes, please ensure all Packages are reconciled correctly and there are no errors. If for some reason you see the UDS Operator throwing errors with `The Client doesn't have the created-by=uds-operator attribute. Rejecting request`, you need to disable the `UDS Client Policy` and give it a bit more time to process all the Packages.
 
+---
+
+In uds-identity-config version 0.11.0 we incorporated some big changes around MFA.
+- Previous versions didn't allow for MFA on the X509 Authentication flow. Now that can be configured to required additional factors of authentication. By default this is disabled and will need to be enabled.
+- Additionally, we've added support of WebAuthn MFA. This can assume many different form such as biometrics, passkeys, etc. This also is disabled by default and is only used as an MFA option.
```

Comment:
```suggestion
- Additionally, we've added support of WebAuthn MFA. This can assume many different forms such as biometrics, passkeys, etc. This also is disabled by default and is only used as an MFA option.
```